src/__init__.py

```diff
@@ -0,0 +1 @@
+"""Pydocstyle - a docstring style checker."""
```

Comment:
Is `src/` meant to be a package? If yes, why? To be clear, I don't think `src/` should be a package.